pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -105,7 +105,7 @@ let
     path = "${cudatoolkit}/lib/stubs/libcuda.so";
   }];
   cudaStubEnv = lib.optionalString cudaSupport
-    "LD_LIBRARY_PATH=${cudaStub}\${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH} ";
+    "LD_LIBRARY_PATH=${cudaStub}\${LD_LIBRARY_PATH:+:}$LD_LIBRARY_PATH ";
```

Comment:
(This one is a no-op just to match the more common style, so that real problems can be found more reliably by grepping for `:$LD_LIBRARY_PATH` and `$LD_LIBRARY_PATH:`.)